lib/hubspot/company_properties.rb

```diff
@@ -19,10 +21,18 @@ def all(opts={}, filter={})
         superclass.all(ALL_PROPERTIES_PATH, opts, filter)
       end
 
+      def get(property_name, opts={})
```

Comment:
shouldn't it be `find` ?
```suggestion
      def find(property_name, opts={})
```